src/logger.ts

```diff
@@ -0,0 +1,3 @@
+export const log = {
+  info: (...args: any[]) => console.log(...args),
+}
```

Comment:
It seems odd to me that calling `log.info` would call `console.log`, shouldn't it call `console.info`?

I'm not too sure about this entire approach, wouldn't it be better to control this via log levels? The test suite could up the log level to `warn` and then we'd use `info` and `debug` inside the code base as appropriate?

Reply:
Yeah, you're totally right. This is a much better approach.